src/styles/navigation.scss

```diff
@@ -0,0 +1,40 @@
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
For both situations considering we create this mixing I am thinking about the problem, it can cause of having unused code for example if I just want to apply display flex and align-items center, In that situation, I will be adding 2 extra properties for every place I will be using flex and same for font group. Thus leading to larger file size for CSS maybe not by a huge margin but with some parts.
Views?...

Reply:
I don't think you will have to repeat the unwanted properties, you can always mentioned the specific argument. Just in case you want to add the align-items center property you can write something like
```scss
@include flex($align-items: center);
```

And it will pick the rest of the properties as default mentioned. Let  me know if I am wrong.

Reply:
https://dev.to/nicolalc/a-mixin-to-rule-them-all-flexbox-driven-layout-mixin-1h43
Found something
I will work on this one in a separate PR right next to the last one I just shared #13

Reply:
https://github.com/divyanshugarg36/brl-cad-web/pull/16
Fixes this

Reply:
Great job :raised_hands: Can you do something similar with the font properties as well? @divyanshugarg36

Reply:
#14  Issue has been created and PR will be sent soon